### PR TITLE
[docs] Fix CLI reference for `ks-prototype`

### PR DESCRIPTION
**Pull request by abiogenesis-now**
@jessicayuen PTAL!

I checked that all the alignments look ok in my terminal. Did some formatting hacks to make different sections of the docs clearer.

Let me know what you think! Am going to try to replicate this across the other commands unless otherwise instructed.

### CODE REVIEW

**Comment by abiogenesis-now** [comment]
Also quick fix because `ks prototype preview --help` wasn't parsing right before

File: cmd/prototype.go [hunk 1]
```diff
@@ -221,16 +220,31 @@ var prototypeSearchCmd = &cobra.Command{
 
 		return nil
 	},
-	Long: `Search ksonnet for prototypes whose names contain` + " `name-substring` " + `.`,
-	Example: `# Search known prototype metadata for the string 'deployment'.
-ks prototype search deployment`,
+	Long: `
+The `+ "`prototype search`" + ` command allows you to search for specific prototypes by name.
```
Comment (jessicayuen) [inline cmd/prototype.go]: can you run `go fmt` on these files?

File: cmd/prototype.go [hunk 2]
```diff
@@ -49,49 +57,25 @@ var prototypeCmd = &cobra.Command{
...
-  --portName=http
-
-# Search known prototype metadata for the string 'deployment'.
-ks prototype search deployment`,
+	Long: `
+Use the` + " `prototype` " + `subcommands to manage, inspect, instantiate, and get
+examples for ksonnet prototypes.
+
+Prototypes are pre-written but incomplete Kubernetes manifests, with "holes"
+(parameters) that can be filled in with the ksonnet CLI. For example, the
```
Comment (jessicayuen) [inline cmd/prototype.go]: `with the ksonnet CLI or manually.`

**Comment by jessicayuen** [comment]
Related: Fixes #179

**Pull request merged**